cirq-core/cirq/ops/clifford_gate_test.py

```diff
@@ -448,7 +448,7 @@ def test_commutes_single_qubit_gate(gate, other):
     itertools.product(_all_clifford_gates(), _paulis, (1.0, 0.25, 0.5, -0.5)),
 )
 def test_commutes_pauli(gate, pauli, half_turns):
-    pauli_gate = pauli ** half_turns
+    pauli_gate = pauli if half_turns == 1 else pauli ** half_turns
```

Comment:
Why is this needed? (pauli ** 1) should be equal to pauli

Reply:
It is surprised to me as well. Actually, they return different type of classes:
```python
print(type(cirq.X), type(cirq.X**1))
# Prints:
# <class 'cirq.ops.pauli_gates._PauliX'> <class 'cirq.ops.common_gates.XPowGate'>
```

Reply:
Let's add a TODO with https://github.com/quantumlib/Cirq/issues/4328 on it for this

Reply:
Done.